src/backend/aspen/api/views/groups.py

```diff
@@ -27,3 +35,35 @@ async def get_group_members(
     group_members_result = await db.execute(group_members_query)
     group_members = group_members_result.scalars().all()
     return GroupMembersResponse.parse_obj({"members": group_members})
+
+
+@router.post("/{group_id}/invitations/", response_model=GroupInvitationsResponse)
+async def invite_group_members(
+    group_id: int,
+    group_invitation_request: GroupInvitationsRequest,
+    auth0_client: Auth0Client = Depends(get_auth0_client),
+    db: AsyncSession = Depends(get_db),
+    settings: Settings = Depends(get_settings),
+    user: User = Depends(get_auth_user),
+) -> GroupInvitationsResponse:
+    if user.group.id != group_id:
+        raise ex.UnauthorizedException("Not authorized")
+    group = (
+        (await db.execute(sa.select(Group).where(Group.id == group_id))).scalars().one()
+    )
+    organization = auth0_client.get_org(group.name)
+    client_id = settings.AUTH0_CLIENT_ID
+    responses = []
+    for email in group_invitation_request.emails:
+        success = True
+        try:
+            auth0_client.invite_member(
+                organization["id"], client_id, user.name, email, "member"
+            )
+        except Auth0Error as err:
+            # TODO - we need to learn more about possible exceptions here.
```

Comment:
Raising an `Auth0Error` looks like this (taken from the library)
````python
raise Auth0Error(
                status_code=self._status_code,
                error_code=self._error_code(),
                message=self._error_message(),
            )
````
which should be enough to make sense of in Sentry. Unless we want to implement more advanced error handling (probably not right now) this should be good enough. The one case I'm concerned about is sending an invite to an email that already has a pending invite, does auth0 handle that gracefully? Will that create another invite or raise an error here?